apps/ui/src/i18n/locale/en/enTranslations.json

```diff
@@ -98,7 +98,8 @@
 		"project": "D. Project Information",
 		"study": "E. Requested Study",
 		"ethics": "F. Ethics",
-		"agreement": "G. Data Access Agreement"
+		"agreement": "G. Data Access Agreement",
+		"appendices": "H. Appendices"
```

Comment:
Can you reorder `E` and `F` as well?
<img width="321" alt="Screenshot 2025-02-14 at 10 59 31 AM" src="https://github.com/user-attachments/assets/2e1a917c-e38f-4324-8e99-3b428e31f0f4" />

Reply:
Thanks for the update!